Bio/PDB/FragmentMapper.py

```diff
@@ -322,15 +322,5 @@ def __getitem__(self, res):
 
 
 if __name__ == "__main__":
-
-    import sys
-
-    p = PDBParser()
-    s = p.get_structure("X", sys.argv[1])
-    m = s[0]
-    fm = FragmentMapper(m, 10, 5, "levitt_data")
-
-    for r in Selection.unfold_entities(m, "R"):
-        print("%s:" % r)
-        if r in fm:
-            print(fm[r])
+    from Bio._utils import run_doctest
```

Comment:
You'd need to expand the existing example in order to make it a self-contained doctest, this alone will fail:

``` python
    >>> model = structure[0]
    >>> fm = FragmentMapper(model, lsize=10, flength=5, dir="fragment_data")
    >>> fragment = fm[residue]
```